server/views/licences/hdc_ap_pss.pug

```diff
@@ -0,0 +1,132 @@
+include includes/licenceMixins
+
+doctype html
+html(lang="en")
+  head
+    link(href= "http://127.0.0.1:3000/public/stylesheets/licences-pdf.css?" + version, media="print", rel="stylesheet", type="text/css")
```

Comment:
Does the IP and port number have to be hard coded here? appreciate there is a lot of legacy code that does the same.